deploy/contents/install/app/install-utils.sh

```diff
@@ -612,25 +612,16 @@ function set_preferences_from_point_in_time_configuration {
     local point_in_time_configuration_preference_file=$(is_module_available_in_point_in_time_configuration system_preference)
     if [ "$point_in_time_configuration_preference_file" ]; then
         print_info "Preferences from point-in-time configuration: ${point_in_time_configuration_preference_file} will be installed."
-        local preferences_file_with_values=$(cat "$point_in_time_configuration_preference_file" | jq -c '[.[] | select(has("value"))]')
-        local payload=$(echo "$preferences_file_with_values" | jq -r '.[] | "\(.name) \(.value) \(.visible)"')
-        while IFS= read -r pref_name pref_value pref_visible; do
-            api_set_preference $pref_name $pref_value $pref_visible
-        done <<< "$payload"
+        local payload=$(jq '[.[] | select(has("value")) | select(.name != ("git.token","cluster.networks.config"))]' $point_in_time_configuration_preference_file)
```

Comment:
to be able to define names that we want to filter you can use the next approach:
```
# regexp like string with all names to filter in format "name1|name2|name3"
PREF_NAMES_TO_FILTER_OUT="git.token|cluster.networks.config"
jq --arg PREF_NAMES_TO_FILTER_OUT "$PREF_NAMES_TO_FILTER_OUT" '[.[] | select(has("value")) | select( .name | test($PREF_NAMES_TO_FILTER_OUT) | not) ]'
```